public/locales/en/translation.json

```diff
@@ -1,42 +1,41 @@
-      "copyAdd": "Anyone you give the link to will be able to get the files. Don't add sensitive or copyrighted files.",
-      "labelSend": "Send the share link",
-      "copySend": "IPFS gives the files a unique fingerprint called a CID that is used to find them again on the network.",
-      "labelKeepPage": "Keep this page open",
-      "labelKeepDaemon": "Keep your IPFS daemon running",
-      "copyKeep": "You are sharing the content from your device. Your friends can help you co-host the content by also keeping the share link open in their browser."
+      "copyAdd": "Add files to share them directly from your device over the IPFS network. Anyone with the share link can download your files, so don't add sensitive or copyrighted material.",
+      "labelKeepPage": "Keep this page open!",
+      "labelKeepDaemon": "Keep your IPFS node running!",
+      "copyKeepPage": "Because you're sharing directly from your device, it's important to keep this page open until the recipient downloads the files. If you're sharing with a group, others in the group can help you co-host the files by also keeping the share page open in their browsers.",
+      "copyKeepDaemon": "Because you're sharing directly from your device, it's important to keep your IPFS node running until the recipient downloads the files. If you're sharing with a group, others in the group can help you co-host the files by keeping the share page open in their browsers.",
+      "labelSend": "Share the link",
+      "copySend": "Every share link contains a unique fingerprint called a CID that enables the recipient's device to find that content on the IPFS network."
     },
     "download": {
-      "title": "Get files directly from other devices",
-      "labelGet": "Get the files",
-      "copyGet": "You can download individual files or, if you want to keep the files structure, click on the download all button.",
-      "labelKeepPage": "Keep this page open",
-      "labelKeepDaemon": "Keep your IPFS daemon running",
-      "copyKeepPage": "You can help co-host the content by also keeping this page open in the browser. This way you are sharing the content from your device.",
-      "copyKeepDaemon": "You can help co-host the content by also keeping your daemon running. This way you are sharing the content from your device.",
-      "labelSend": "Send the share link",
-      "copySend": "IPFS gives the files a unique fingerprint called a CID that is used to find them again on the network."
-    },
+      "title": "You've got files",
+      "subtitle": "Download directly from the sender — no cloud needed!",
+      "labelHow": "How does IPFS Share work?",
+      "copyHow": "IPFS Share uses the global IPFS network to enable people to share files directly with each other — no cloud or third-party server needed. And because files on IPFS are identified by what they contain, not where they're located, you can be sure what you download is the same content that the sender intended.",
```

Comment:
Nit, but would recommend using "that" either in both parts or neither part here... 
"you can be sure THAT what you download is the same content that the sender intended." or 
"you can be sure what you download is the same content ~that~ the sender intended.

Reply:
Good call, I'll remove the second "that". Was getting a little late at night by this point.

```suggestion
      "copyHow": "IPFS Share uses the global IPFS network to enable people to share files directly with each other — no cloud or third-party server needed. And because files on IPFS are identified by what they contain, not where they're located, you can be sure what you download is the same content the sender intended.",
```